src/applications/vaos/project-cheetah/components/SelectDate1Page.jsx

```diff
@@ -98,19 +98,22 @@ export function SelectDate1Page({
     appointmentSlotsStatus === FETCH_STATUS.notStarted;
   const isInitialLoad = useIsInitialLoad(loadingSlots);
 
-  useEffect(() => {
-    getAppointmentSlots(
-      moment()
-        .startOf('month')
-        .format('YYYY-MM-DD'),
-      moment()
-        .add(1, 'months')
-        .endOf('month')
-        .format('YYYY-MM-DD'),
-      true,
-    );
-    document.title = `${pageTitle} | Veterans Affairs`;
-  }, []);
+  useEffect(
+    () => {
+      getAppointmentSlots(
+        moment()
+          .startOf('month')
+          .format('YYYY-MM-DD'),
+        moment()
+          .add(1, 'months')
+          .endOf('month')
+          .format('YYYY-MM-DD'),
+        true,
+      );
+      document.title = `${pageTitle} | Veterans Affairs`;
+    },
+    [getAppointmentSlots],
```

Comment:
I've generally not been putting these action creators in the dependencies arrays for effects that are meant to run once when the component is first rendered, even though the linter warns about it. I think the empty array is a clearer indication of when we intend the effect to run, and it also protects us if the function is redefined unexpectedly (which shouldn't happen for `getAppointmentSlots`, but could happen if we're just passing down some random function prop).

Reply:
That makes sense. I think it's clearer if we maintain that pattern as well.